presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -751,6 +751,30 @@ so a cross join between the two tables produces 125 rows::
     ...
     (125 rows)
 
+LATERAL
+^^^^^^^
+
+Subqueries appearing in the ``FROM`` clause can be preceded by the keyword ``LATERAL```.
+This allows them to reference columns provided by preceding FROM items.
+
+A ``LATERAL`` join can appear at the top level in the ``FROM`` list, or anywhere
+within a parenthesized join tree. In the latter case, it can also refer to any items
+that are on the left-hand side of a ``JOIN`` that it is on the right-hand side of.
+
+When a ``FROM`` item contains ``LATERAL`` cross-references, evaluation proceeds as follows:
+for each row of the FROM item providing the cross-referenced column(s),
+the ``LATERAL`` item is evaluated using that row or row set's values of the columns.
+The resulting row(s) are joined as usual with the rows they were computed from.
+This is repeated for each row or set of rows from the column source table(s).
+
+``LATERAL`` is primarily useful when the cross-referenced column is necessary for
+computing the row(s) to be joined::
```

Comment:
Is this "row(s)" because a query might only have one row? If so, I think it's less confusing to always say "rows"